docs/pages/includes/install-agent-cloud.mdx

```diff
@@ -1,5 +1,7 @@
-On the host where you will run your Teleport Node, install a version of
-Teleport that is compatible with the Teleport Cloud version, v(=cloud.version=):
+{{ service="your Teleport service" }}
+
+On the host where you will run {{ service }}, install a version of Teleport that
+is compatible with the Teleport Cloud version, v(=cloud.version=):
```

Comment:
Do we have a page we can link to here that shows version compatibility between different Teleport versions?